packages/mobile/src/components/scrubber/usePosition.ts

```diff
@@ -47,6 +50,16 @@ export const usePosition = (
     }
   }, [isPlaying, isInteracting, duration])
 
+  // Android pauses timeouts when in background, so we use app state
+  // to trigger a position coercion
+  const appState = useAppState()
+
+  useAsync(async () => {
```

Comment:
Could we `useEnterForeground` instead?

Reply:
yeeee

Reply:
interesting, so looks like useEnterForeground's deps and rerenders are a bit off, since itll trigger everytime incoming function is remade... we might want to add a dep array to that impl, or pass useCallbacks into it? wonder what you think is best here...

Reply:
going to merge for now, but im keen to update useEnterForeground etc to be top perf, and then integrate that here.